docs/components/board/jetson.md

```diff
@@ -44,10 +44,21 @@ Edit and fill in the attributes as applicable.
 {
   "components": [
     {
-      "name": "<your-jetson-board>",
-      "type": "board",
+      "name": "<your-jetson-board-name>",
       "model": "jetson",
-      "attributes": {},
+      "type": "board",
+      "namespace": "rdk",
+      "attributes": {
+        "digital_interrupts": [
+          <...See table below...>
+        ],
+        "spis": [
```

Comment:
we can either get ahead of removing spi and i2c attributes or wait until they're merged in for generic linux/custom linux drivers, up to you - cc @penguinland @oliviamiller